src/index.js

```diff
@@ -4,10 +4,31 @@ import './wdyr';
 
 import React from 'react';
 import ReactDOM from 'react-dom';
+import * as Sentry from '@sentry/react';
```

Comment:
Can you make import only if Sentry is used (i.e. REACT_APP_SENTRY_DSN var is not undefined)

Reply:
wow i didn't know it was possible, i try it right now 🙏

Reply:
Done 👍